src/Hooks.res

```diff
@@ -36,6 +36,7 @@ module Form = {
     clearErrors: (. string) => unit,
     setError: (. string, Error.t) => unit,
     reset: (~defaultValues: Js.Dict.t<Value.t>=?, unit) => unit,
+    getValues: (~fieldName: string) => string,
```

Comment:
Sorry Thien, this will not always return a string, it can return any valid JS Object, so the type should be `option<Js.Json.t>` and we need some decoders here.

Also we could add `getAllValues(~fieldNames: array<string>): array<option<Js.Json.t>>` as the original function can accept an array 👍

Reply:
@gaku-sei -san. Sorry i might miss sth but I get the one when try to return type option<...> 
<img width="566" alt="Hooks_res_—_rescript-hook-form" src="https://user-images.githubusercontent.com/30365498/119296525-0f21ff00-bc94-11eb-98b5-c3d4156d944e.png"> 🙇 

Also you meant get another one called `getAllValues` side by side with `getValues` Kevin-san?

Actually the: getValues in react-hook-form are quite flexible 

`getValues(name: string | array<string>|undefined) => object`
_ only string name: 1 return data for that field
_ array string field names: return object that contains all fields
_ undefined: obj all values in the form

I tried to achieve it but cannot even I just tried achieve 1 and 3 which I used to define
```
getValues: (~fieldName: string=?,()) => string,
```
but somehow when using it getValues() => it doesnt return me the obj that contain whole form